tests/library/trace-viewer.spec.ts

```diff
@@ -30,6 +30,10 @@ test.slow();
 
 let traceFile: string;
 
+test.use({
+  baseURL: 'https://example.com',
```

Comment:
Instead of affecting all tests, let's pass `baseURL` option to the `newContext()` call on the line 38.

Reply:
I somehow thought that wouldn't work, but it did. done in https://github.com/microsoft/playwright/pull/31852/commits/d0eb8056554323735b6129dacb3ea176431ee4b4